doc/source/whatsnew/v0.24.0.txt

```diff
@@ -479,6 +479,7 @@ all-``NaT``.  This is for compatibility with ``TimedeltaIndex`` and
 Previous Behavior:
 
 .. code-block:: ipython
+- Comparing :class:`Timedelta` with unknown types now return NotImplemented instead of False (:issue:`20829`)
```

Comment:
use double backticks around NotIMplemented and False.

this note is in an odd place